src/layer/layers-manager.js

```diff
@@ -56,7 +60,7 @@ export default class LayersManager extends React.Component<LayersManagerPropsT>
                 zIndex: this.props.zIndex,
               }}
             >
-              <AppContainer {...appContainerProps}>
+              <AppContainer {...appContainerProps} ref={this.containerRef}>
```

Comment:
This will only apply to baseui elements that are not portaled right? Can we move it up a level or apply the polyfill to the layers container as well?

Reply:
so this ref doesn't need to be a parent of component that uses `focusVisible` logic, we just need to get `node.ownerDocument` to attach some global event listeners like `mousedown`... so I was looking for a node that always gets mounted and this should be it?